x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/set_message_helpers.ts

```diff
@@ -38,3 +40,17 @@ export const setQueuedErrorMessage = (message: string) => {
 export const clearFlashMessages = () => {
   FlashMessagesLogic.actions.clearFlashMessages();
 };
+
+export const flashSuccessToast = (toast: IToast) => {
+  FlashMessagesLogic.actions.addToastMessage({
+    ...FLASH_MESSAGE_TYPES.success,
+    ...toast,
+  });
+};
+
+export const flashErrorToast = (toast: IToast) => {
```

Comment:
Along with above I would personally prefer `addErrorToast` to avoid any potential confusion with `flashMessages`

Reply:
Hmm, this is super interesting to me! So for me conceptually, when I was thinking about these changes to the FlashMessages component, I was like: "A flash message is a generic idea! The main concept is that it's a message, but in the UI it can present as either a callout OR a toast (but is a callout by default for historical reasons)" - and I wanted to kind of highlight that connection with this fn name.

Coincidentally, I originally wanted to pitch renaming all the other helpers from `setX` to `flashX` to you - e.g. `flashSuccessMessage`, `flashErrorMessage`, `queueSuccessMessage` - but thought that was too much for this PR 😆

Not sure if that concept sparks any joy with you at all - would love thoughts from @JasonStoltz and @byronhulcher as well. One thing I really like about the idea of a "flash message" as a generic concept is that we could continue to expand this component to include more types of alerts - e.g. it could possibly even contain `EuiNotificationEvents` if it had to (as an example, I don't think we would in reality).

Reply:
Just to close the loop on this naming convo after chatting briefly w/ Scotty in Slack, we're going to leave this as `flashSuccessToast` and `flashErrorToast` in this PR. In a follow-up PR that I'll open up right after this one, we will then change the other helpers in this file to:

`setSuccessMessage` -> `flashSuccessCallout`
`setErrorMessage` -> `flashErrorCallout`
`setQueuedSuccessMessage` -> `queueSuccessCallout`
`setQueuedErrorMessage` -> `queueErrorCallout`

I'm opting to make those changes in a separate PR so I don't add a ton of extra file diffs to this PR right now. 🙇‍♀️